tests/client.c

```diff
@@ -95,7 +91,7 @@ enum demo_result
 do_read(struct conndata *conn, struct rustls_connection *rconn)
 {
   int err = 1;
-  int result = 1;
```

Comment:
oops: looks like I accidentally squashed in some of the unchecked conversion fixes into this header fix commit.

I'll untangle these on monday.

Reply:
Fixed up.